README.md

```diff
@@ -1,45 +1,112 @@
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
 # Football Players Statistics WebCrawler
 
 This project is a sub-module for [Multiplayer Football Draft Simulator](https://github.com/sauravhiremath/fifa).
 
 # About
 
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
```

Comment:
I think all badges in one line (preferably at the top) will be more concise and informative. This also helps in better UX for the readme

Reply:
Yeap that indeed looks better than before! Added a couple of more badges! and I have moved it to the top. The other render issue was due to improper line breaks. <br/> tag fixed it instead of <br>.
Hopw this time everything is okay!